### PR TITLE
Data manager post fix

### PR DESCRIPTION
**Pull request by akiskips**

### CODE REVIEW

**Comment by Manu3756** [comment]
> Issue https://sapjira.wdf.sap.corp/browse/I40AASSERVICE-164 is still broken for me when I run this branch and do the configuration of endpoint- and apater-registry with the Postman Collection from this branch.

I think this is related to the wrong ID ".../mongo" of the registered submodel-routing.

**Comment by akiskips** [comment]
The POST request registering the submodel - adapter assignmene  for the adapter registry was updated. @Manu3756 I removed the /mongo from the submodel id, was added initially for debugging, in order to know if the submodel was routed correctly

File: src/compose/volumes/initializer/registerEndpoints.json [hunk 1]
```diff
@@ -1,26 +1,56 @@
...
+        }
+    },
+    {
+        "aasId": {
+            "id": "https://i40-aas-approval-app.cfapps.eu10.hana.ondemand.com/aas",
+            "idType": "IRI"
+        },
+        "endpoints": [
+            {
+                "url": "https://admin:admin123@i40-aas-approval-app.cfapps.eu10.hana.ondemand.com/register",
```
Comment (etoster) [inline src/compose/volumes/initializer/registerEndpoints.json]: What is our policy on publishing the credentials to our CF app? @Manu3756 @kasimrehman 
We agreed not to do this for now, didn't we?

Comment (akiskips) [reply src/compose/volumes/initializer/registerEndpoints.json]: Indeed, the json should not be in the version control and was updated by mistake.
I changed it back to the version in canary

**Pull request merged**